aspnet/mvc/views/overview.rst

```diff
@@ -0,0 +1,79 @@
+Views Overview
+==============
+
+By `Steve Smith`_
+
+ASP.NET MVC Core controllers can return formatted results using *views*.
+
+.. contents:: Sections
+  :local:
+  :depth: 1
+
+What are Views?
+---------------
+
+In the Model-View-Controller (MVC) pattern, the *view* encapsulates the presentation details of the user's interaction with the app. Views are templates that include HTML with embedded code that generates content to send to the client. The code uses :doc:`razor syntax <razor>`, which have deep understanding of HTML that allows it to interact with the HTML in the document with minimal code or ceremony.
```

Comment:
> The code uses :doc:`razor syntax <razor>`, which have deep understanding of HTML that allows it to interact with the HTML in the document with minimal code or ceremony

Views use :doc:`Razor syntax`, which allows code to interact with HTML with minimal code or ceremony.